test/functional/sandbox/html/bogusCommand.dev.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Alloy Test Page</title>
+        <script>
+            !function(n,o){o.forEach(function(o){n[o]||((n.__alloyNS=n.__alloyNS||
+            []).push(o),n[o]=function(){var u=arguments;return new Promise(
+            function(i,l){n[o].q.push([i,l,u])})},n[o].q=[])})}
+            (window,["alloy"]);
+        </script>
+        <script src="/dist/standalone/alloy.js" async></script>
+
+        <script>
+            alloy("configure", {
```

Comment:
If logging isn't enabled using the logEnabled configuration, Testcafe is unable to fetch the browser console logs.

Reply:
If you add "?alloy_log=true" to the test urls it will ensure logging is enabled before the configure command is called so you get log output for the config command being called.

Reply:
Also I like that you used the word bogus. 👍 👍 👍

Reply:
Adding a separate test spec with the url flag to see if getBrowserConsoleMessages is able to retrieve the logs.